NOTICE

```diff
@@ -52,7 +52,7 @@ Copyright jQuery Foundation and other contributors
 Licensed under the MIT License
 
 Go support for Protocol Buffers - Google's data interchange format
-http://github.com/golang/protobuf/
+http://github.com/gogo/protobuf/
```

Comment:
The gogo/protobuf has a different LICENSE text. It needs to be updated here.

Reply:
if you can please go ahead and makes these changes.
or let me know what should I change.

I don't even know what this NOTICE document is for :)

Reply:
It's a requirement of the Apache license.

Will update the license myself. Merging now...

Reply:
In different news, NOTICE is horribly out of date anyway. But at least we don't want plainly wrong copyright statements there. Already fixed in master.